OracleWebCenterContent/dockerfiles/README.md

```diff
@@ -0,0 +1,274 @@
+```
+docker stop container WCCAdminContainer
+```
+
+#### C. Starting Admin Container
+```
+docker start container -i WCCAdminContainer
+```
+
+### 5.1.3. Managed Server Container (WCContentContainer)
+
+#### A. Creating and Running WCContent Container
+Run the following command to create the WCContent Managed Server container:
+
+```
+docker run -it --name WCContentContainer --network=WCContentNET -p <UCM_HOST_PORT>:<UCM_PORT> -p <IBR_HOST_PORT>:<IBR_PORT> -p <UCM_INTRADOC_PORT>:<UCM_INTRADOC_PORT> -p <IBR_INTRADOC_PORT>:<IBR_INTRADOC_PORT> --volumes-from WCCAdminContainer --env-file <PATH_TO_ENV_FILE>/webcenter.env.list oracle/wccontent:12.2.1.4.0 configureOrStartWebCenterContent.sh
+
+# A sample command will look like this -
+
+docker run -it --name WCContentContainer --network=WCContentNET -p 16200:16200 -p 16250:16250 -p 4444:4444 -p 5555:5555 --volumes-from WCCAdminContainer --env-file <PATH_TO_ENV_FILE>/webcenter.env.list oracle/wccontent:12.2.1.4.0 configureOrStartWebCenterContent.sh
+```
+The docker run command creates the container as well as starts the WebCenter Content managed servers. 
+
+Note: 1. If manged servers need to be accessed through host ports different from container ports, then intended host port values needs to be supplied as part of -p option of the `docker run`
+command mentoned above (for ex. -p 16201:16200 and -p 16251:16250). The same port value needs to be updated in the `webcenter.env.list` as `UCM_HOST_PORT` and `IBR_HOST_PORT`.
+If managed servers are going to be accessed via same host port number as the container port, then `UCM_PORT` and `UCM_HOST_PORT` values (and `IBR_PORT` and `IBR_HOST_PORT`) should be same in the `webcenter.env.list`.  
+      2. Intradoc ports are for internal server communications and not meant for browser access. While, intradoc ports on container are configurable (like other parametres like admin credentials, admin port, domain-name, manged server container ports) 
+through `webcenter.env.list`, publishing these to different host ports is not supported. This essentially means one can provide `-p 7777:7777` instead of `-p 4444:4444`, but `-p 6666:7777` is not supported.
+
+#### B. Stopping WCContent Container
+```
+docker container stop WCContentContainer
+```
+
+#### C. Starting WCContent Container
+```
+docker container start -i WCContentContainer
+```
+
+#### D. Getting Shell in WCContent Container
+```
+docker exec -it WCContentContainer /bin/bash
+```
+Please wait till all the above all `docker run` commands are successfully completed, before you can start Admin and Managed Servers with WebLogic admin credentials.
+
+WebLogic Admin Server
+http://hostname:7001/console/
+
+UCM Server
+http://hostname:16200/cs/
+
+IBR Server
+http://hostname:16250/ibr/
+
+# 6. License
+
+To download and run Oracle Fusion Middleware, regardless whether inside or outside a container, you must download the binaries from the Oracle website and accept the license indicated at that page.
```

Comment:
This should refer to the Oracle Software Delivery Cloud not "the Oracle website"